src/metrics/instance.rs

```diff
@@ -0,0 +1,62 @@
+//! This module defines all the instance-level metrics of crates.io.
+//!
+//! Instance-level metrics are collected separately for each instance of the crates.io application,
+//! and are then aggregated at the Prometheus level. They're not suited for service-level metrics
+//! (like "how many users are there").
+//!
+//! There are two ways to update instance-level metrics:
+//!
+//! * Continuously as things happen in the instance: every time something worth recording happens
+//!   the application updates the value of the metrics, accessing the metric through
+//!   `req.app().instance_metrics.$metric_name`.
+//!
+//! * When metrics are scraped by Prometheus: every tot seconds Prometheus sends a request to the
```

Comment:
I'm not sure what "tot seconds" is in reference to.

Reply:
It just means "every `N` seconds". Did I accidentally use an Italian-ism here? :sweat_smile: